utils/Makefile

```diff
@@ -20,6 +20,7 @@ CFLAGS += -fdata-sections
 CFLAGS += -Wno-conversion
 CFLAGS += -Wno-parentheses
 CFLAGS += -Wstack-usage=512
+CFLAGS += -O3
```

Comment:
As discussed, we should not use `-O3` for debug builds.

Reply:
these kind of flags should also be in the main top-level options that other projects share. Having every project have its own options like this is a real pain!

Reply:
Removed. I left it in by accident.